component/src/main/java/com/synopsys/integration/alert/component/settings/environment/ProxySettingsEnvironmentHandlerFactory.java

```diff
@@ -57,12 +58,13 @@ public EnvironmentVariableHandler build() {
     }
 
     private Boolean isConfigurationMissing() {
-        return configAccessor.getConfiguration().isPresent();
+        return configAccessor.getConfiguration().isEmpty();
```

Comment:
Would this be a case for the exists method?

Reply:
the exists method for SettingsProxyConfigAccessor wasn't created until 6.10.0. My idea was when merging into master  to update this in master to use the exists method.